### PR TITLE
Improved mutation score

### PR DESCRIPTION
**Pull request by kukulich**

### CODE REVIEW

File: src/NodeCompiler/CompileNodeToValue.php [hunk 1]
```diff
@@ -25,6 +25,8 @@
  */
 class CompileNodeToValue
 {
+    private const TRUE_FALSE_NULL = ['true', 'false', 'null'];
```
Comment (Ocramius) [inline src/NodeCompiler/CompileNodeToValue.php:28]: Cheeky way to avoid mutation tests :D

**Review by Ocramius — APPROVED** [review]
LGTM! :ship: 

@kukulich still working on this, or mergeable?

**Comment by kukulich** [comment]
@Ocramius Just forcepushed my actual version. I can open another PR when there's anything more to fix.

File: src/NodeCompiler/CompileNodeToValue.php (hunk 1 above)
Comment (kukulich) [reply src/NodeCompiler/CompileNodeToValue.php:28]: Yes, naming is hard... :)

File: src/Reflection/Adapter/ReflectionFunction.php [hunk 2]
```diff
@@ -183,7 +183,7 @@ public function invoke(mixed ...$args): mixed
         try {
             return $this->betterReflectionFunction->invoke(...func_get_args());
         } catch (Throwable $e) {
-            throw new CoreReflectionException($e->getMessage(), 0, $e);
+            throw new CoreReflectionException($e->getMessage(), previous: $e);
```
Comment (Ocramius) [inline src/Reflection/Adapter/ReflectionFunction.php:186]: Ugly but 🤷‍♀️

**Review by Ocramius — APPROVED** [review]

**Comment by Ocramius** [comment]
Thanks @kukulich!

**Pull request merged**